src/cljc/chronograph/specs.cljc

```diff
@@ -59,3 +59,42 @@
 (s/def :tasks/description string?)
 (s/def :tasks/create-params-handler (s/keys :req-un [:tasks/name]
                                             :opt-un [:tasks/description]))
+
+
+;; Timers - DB
+
+
+(s/def :timers/id uuid?)
+(s/def :timers/user-id :users/id)
+(s/def :timers/task-id :tasks/id)
+(s/def :timers/note string?)
+
+(s/def :timers/timer (s/keys :req [:timers/id :timers/user-id :timers/task-id]
+                             :opt [:timers/note]))
+
+(s/def :handlers.timer/create-request-body (s/keys :req-un [:timers/task-id]
+                                                   :opt-un [:timers/note]))
+
+
+;; Time Spans - DB
+
+
+(s/def :time-spans/id uuid?)
+(s/def :time-spans/timer-id :timers/id)
+
+(s/def :time-spans/time-span (s/keys :req [:time-spans/id :time-spans/timer-id]))
```

Comment:
Is this missing `started-at` and `stopped-at`?

Reply:
Added these